lib/task/import/validate/csvBaseValidator.class.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * CSV validation test base class. All test classes in lib/task/import/validate
+ * extends this class. Each test class should override:
+ * 1) __construct() - set title and any other test initialization
+ * 2) testRow() - Called once per row in the CSV. Test row and record results here.
+ * 3) reset() - Reset variables as needed before processing next CSV file.
+ * 4) getTestResults() - finalize test results and set details in $testData (CsvValidatorResult).
+ *
+ * See class CsvSampleValuesValidator for an example.
+ *
+ * @author     Steve Breker <sbreker@artefactual.com>
+ */
+abstract class CsvBaseValidator
+{
+    const HEADER_PLACEHOLDER = 'EXTRA_COLUMN';
+
+    protected $filename = '';
+    protected $columnCount = 0;
+    protected $rowNumber = 1;
+    protected $title = '';
+    protected $options = [];
+    protected $ormClasses = [];
+    protected $testData;
+
+    public function __construct(array $options = null)
+    {
+        if (isset($options)) {
+            $this->setOptions($options);
+        }
+
+        $this->testData = new CsvValidatorResult($this->title, $this->filename, $this->getClassName(), $this->getVerbose());
+    }
+
+    public function testRow(array $header, array $row)
+    {
+        ++$this->rowNumber;
+    }
+
+    public function reset()
+    {
+        $this->filename = '';
+        $this->testData = new CsvValidatorResult($this->title, $this->filename, $this->getClassName(), $this->getVerbose());
+    }
+
+    public function getVerbose()
+    {
+        return (isset($this->options['verbose']) && $this->options['verbose']) ? true : false;
```

Comment:
This can be simplified to:

```
return (isset($this->options['verbose']) && $this->options['verbose']);
```

Reply:
And further to `return !empty($this->options['verbose'])`.

Reply:
This code was removed completely during a refactor to remove the unused 'verbose' option from the validations.